models/dex/dex_trades.sql

```diff
@@ -65,6 +65,7 @@
 ,ref('camelot_trades')
 ,ref('zeroex_native_trades')
 ,ref('xchange_trades')
+,ref('honeyswap_v2_trades')
```

Comment:
Please add your model to https://github.com/duneanalytics/spellbook/blob/main/models/dex/dex_trades_migration_beta.sql instead. It's temporary and will replace dex_trades when it's done.

Reply:
added in https://github.com/duneanalytics/spellbook/pull/4139/commits/d5e9e9f8bc528af20a2e75aa0962afc2d6aefea0